### PR TITLE
Pigeon maps: draw map and vehicle markers

### PR DESCRIPTION
**Pull request by lemald**
Asana ticket: [⚙️ Pigeon replacement: Draw map, icons, and labels (1/3)](https://app.asana.com/0/1200180014510248/1200542106910651/f)

### CODE REVIEW

**Comment by lemald** [comment]
@phildarnowsky I'm probably going to polish some pieces of this further, but for the most part it's generally ready for review.

File: assets/src/components/map.tsx [hunk 1]
```diff
@@ -107,7 +153,62 @@ const makeLabelIcon = (
...
+          y="50%"
+          textAnchor="middle"
+          dominantBaseline="central"
+        >
+          {labelString}
+        </text>
+      </svg>
+    </div>
+  )
+  // iconAnchor: [labelBackgroundWidth / 2, isPrimary ? -16 : -10],
```
Comment (phildarnowsky) [inline assets/src/components/map.tsx:208]: Commented-out line here

File: assets/src/components/map.tsx [hunk 2]
```diff
@@ -338,20 +504,71 @@ const Map = (props: Props): ReactElement<HTMLDivElement> => {
...
       </ReactLeafletMap>
     </>
   )
 }
 
+const mapTiler = (x: number, y: number, z: number): string =>
+  `https://mbta-map-tiles-dev.s3.amazonaws.com/osm_tiles/${z}/${x}/${y}.png`
+
+const MapWithPigeon = (props: Props): ReactElement<HTMLDivElement> => {
+  /* const latLngs: [number, number][] = props.vehicles.map(
```
Comment (phildarnowsky) [inline assets/src/components/map.tsx:534]: more commented-out code

**Review by phildarnowsky — APPROVED** [review]
looks good apart from some commented-out code that can come out

**Pull request merged**